pkg/components/state/pluggable.go

```diff
@@ -321,6 +322,18 @@ func (ss *grpcStateStore) Multi(ctx context.Context, request *state.Transactiona
 	return err
 }
 
+// MultiMaxSize returns the maximum number of operations allowed in a transactional request.
+func (ss *grpcStateStore) MultiMaxSize() int {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	resp, err := ss.Client.MultiMaxSize(ctx, new(proto.MultiMaxSizeRequest))
+	if err != nil {
+		return -1
+	}
+	return int(resp.GetMaxSize())
```

Comment:
Needs overflow check on 32-bit systems

Reply:
This would happen if the pluggable component runs on a 64-bit machine and dapr/dapr on a 32-bit one. It's probably never going to happen in reality, or at least I don't know of any databases that have a limitation of a few billion transactions, but it's always nice to be precise. 
So how should we handle this? Should we limit it to `math.MaxInt32`?

Reply:
Correct, it is unlikely but still _possible_

I think a simple way would be to check if it's higher than math.MaxInt32, then return an error?

Reply:
So that's the thing, it's the same issue as above; we're bound by the interface and can't return am error. So maybe just log it, as you suggested above, and return math.MaxInt32?